docs/source/contents/examples/batch-examples-local.md

```diff
@@ -0,0 +1,648 @@
+
+```bash
+git clone https://github.com/SeldonIO/seldon-core --branch=v2
+cd seldon-core/
+```
+
+You can then start Seldon Core locally by running:
+
+```bash
+metadata:
+  name: iris
+make deploy-local
+```
+
+```{important}
+By default, the CLI will expect your inference endpoint to be at `0.0.0.0:9000`. If you have customized this, you'll need to [redirect the CLI](../cli/index).
+```
+
+## Deploy Models and Pipelines
+
+First, let's jump in to the `samples` folder where we'll find some sample models and pipelines we can use:
+
+```bash
+cd samples/
+```
+
+### Deploy the Iris Model
+
+Let's take a look at a sample model before we deploy it:
+
+```bash
+cat models/sklearn-iris-gs.yaml
+```
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Model
+metadata:
+  name: iris
+spec:
+  storageUri: "gs://seldon-models/mlserver/iris"
+  requirements:
+  - sklearn
+  memory: 100Ki
+```
+
+The above manifest will deploy a simple [sci-kit learn](https://scikit-learn.org/stable/) model based on the [iris dataset](https://archive.ics.uci.edu/ml/datasets/iris).
+
+Let's now deploy that model using the Seldon CLI:
+
+```bash
+seldon model load -f models/sklearn-iris-gs.yaml
+```
+
+### Deploy the Iris Pipeline
+
+Now that we've deployed our iris model, let's create a [pipeline](../pipelines/index) around the model. 
+
+```bash
+cat pipelines/iris.yaml
+```
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: iris-pipeline
+spec:
+  steps:
+    - name: iris
+  output:
+    steps:
+    - iris
+```
+
+We see that this pipeline only has one step, which is to call the `iris` model we deployed earlier. We can create the pipeline by running:
+
+```bash
+seldon pipeline load -f pipelines/iris.yaml
+```
+
+### Deploy the Tensorflow Model
+
+To demonstrate batch inference requests to different types of models, we'll also deploy a simple [tensorflow](https://www.tensorflow.org/) model:
+
+```bash
+cat models/tfsimple1.yaml
+```
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Model
+metadata:
+  name: tfsimple1
+spec:
+  storageUri: "gs://seldon-models/triton/simple"
+  requirements:
+  - tensorflow
+  memory: 100Ki
+
+```
+
+The tensorflow model takes two arrays as inputs and returns two arrays as outputs. The first output is the addition of the two inputs and the second output is the value of (first input - second input).
+
+Let's deploy the model:
+
+```bash
+seldon model load -f models/tfsimple1.yaml
+```
+
+### Deploy the Tensorflow Pipeline
+
+Just as we did for the scikit-learn model, we'll deploy a simple pipeline for our tensorflow model:
+
+Inspect the pipeline manifest:
+```bash
+cat pipelines/tfsimple.yaml
+```
+```yaml
+apiVersion: mlops.seldon.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: tfsimple
+spec:
+  steps:
+    - name: tfsimple1
+  output:
+    steps:
+    - tfsimple1
+
+```
+and deploy it:
+```bash
+seldon pipeline load -f pipelines/tfsimple.yaml
```

Comment:
You could use the seldon CLI to wait for model or pipeline ready as done in other examples?

Reply:
I quite wanted to show the `model list` and `pipeline list` functions of the CLI though, which I do underneath. That way people get a variety of different ways to see things across the examples.